app/views/api/v2/hosts/base.json.rabl

```diff
@@ -1,3 +1,7 @@
 object @host
 
 attributes :name, :id
+
```

Comment:
In new host detail page it's showing full name, even when the settings is `false`
![name](https://github.com/theforeman/foreman/assets/59385976/5e64d08d-ddc0-465e-91ac-4cae8907cd92)

Reply:
Do you have this PR checked out? https://github.com/Katello/katello/pull/10538

Reply:
I was testing it on instance without Katello

Reply:
Thanks for pointing this out. Fixed.